### DISCUSSION TITLE
Is it possible to show (dynamic) custom text on a Splash Screen?

### DISCUSSION THREAD

**Discussion by bchehraz** [ionic-team/capacitor#5298] [Q&A]
I'm looking to put a user's company name on the splash screen which is non-static data. Only once it's in app storage would it display, but I just wanted to know if this were possible. From my brief research, I couldn't find anything on whether this is even possible.

On the web version of the app I'm able to briefly show a loader on initialization and show their company name, but because I'm using the splash screen on the mobile app,... I could show a loader AFTER the splash screen but this would kind of defeat the purpose of the splash screen.

Any ideas on how I could do this? Thanks!

**Comment by angeloraso** [comment]
Hi. What I do is use a 2732x2732 image where I show the company logo and its name


![splash](https://user-images.githubusercontent.com/15707453/145082739-9b689a05-2516-4732-aca6-3f223c609978.png)

**Reply by bchehraz** [reply]
In your implementation, how do you pull the company information into the image?

Is it a generic image that you can then replace with company information at any time without doing a re-build?

In my particular use case, the company name can change at any time while the user is on the app.

**Reply by angeloraso** [reply]
- It's a static asset. We include it within the project.
- Sorry. It seems that this solution will not work for you. I have not heard an alternative for that specific issue

**Reply by bchehraz** [reply]
I see... No problem, thanks anyway  😄
